client/state/utils/index.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Internal dependencies
+ */
+import { cachingActionCreatorFactory } from './caching-action-creator-factory';
+import { createReducer } from './create-reducer';
+import { extendAction } from './extend-action';
+import { isValidStateWithSchema, withSchemaValidation } from './schema-utils';
+import { keyedReducer } from './keyed-reducer';
+import { withEnhancers } from './with-enhancers';
+import { withoutPersistence } from './without-persistence';
+import { withStorageKey } from './with-storage-key';
+import { addReducer, combineReducers } from './reducer-utils';
+
+export {
+	addReducer,
+	cachingActionCreatorFactory,
+	combineReducers,
+	createReducer,
+	extendAction,
+	isValidStateWithSchema,
+	keyedReducer,
+	withEnhancers,
+	withoutPersistence,
+	withSchemaValidation,
+	withStorageKey,
+};
```

Comment:
Why not re-export them individually? I.e. 

```js
export { cachingActionCreatorFactory } from './caching-action-creator-factory';
```

etc

Reply:
Done in 6125f71de7 :grimacing:

Reply:
👍 Thanks!